src/Personal/login.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import {
+  View, Text, TextInput, Button,
+} from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import utils from './utils';
+import LoginView from './loginView';
+import { requestLogin } from './actionCreator';
+
+export default function Login() {
+  const { navigate } = useNavigation();
+  const user = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const login = (userInfo) => {
+    dispatch(requestLogin(userInfo.email, userInfo.password));
```

Comment:
You can just pass userInfo into the function, as I noticed in the POST method, you passed the combined email and password variables again